travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
```

Comment:
I think we moved to github actions

Reply:
yes, no one told me when doing this update. will remove